docs/ref/contrib/postgres/fields.txt

```diff
@@ -19,6 +19,43 @@ may be a good choice for the :ref:`range fields <range-fields>` and
 :class:`HStoreField`, and GIN may be helpful for :class:`ArrayField` and
 :class:`JSONField`.
 
+``SerialField``
+===============
+
+.. versionadded:: 2.2
```

Comment:
`.. versionadded:: 3.0`

Reply:
Fields are documented in alphabetical order. I would also suggest combining these into a single ` ``SerialField``s ` heading, similar to ` ``CIText`` fields `.